05-history.md

```diff
@@ -202,6 +202,25 @@ on the other hand,
 moving backward and forward in time becomes much easier.
 
 
+
+> ## Checking out changed files {.challenge}
+>
+> Assume you have started to work on something, like above and git says to discard thing.
+> How can the command above "git checkout -- mars.txt" be described in terms of git?
+>
+> a) take the last snapshot and overwrite my pending changes
+> b) in the history of the repository, go back one step
+> c) clear the staging area from the changes I did to mars.txt
+> d) delete everything I have done since I last committed
+>
```

Comment:
Possible rewording:

a)  takes the last commit (snapshot) and reverses any changes made in the working area since that commit.
b)  goes back one step in the history of the repository  (can test for understanding of *where HEAD is* v. *moving HEAD around*)
c)  clears the staging area of any changes to `mars.txt`    (a v. c tests for orthogonality of *things under version control* and *things which happen to share the same folder but aren't all necessarily under version control*)

Is a different from d?